atomicapp/providers/docker.py

```diff
@@ -90,6 +90,8 @@ def run(self):
             label_run = None
             with open(artifact_path, "r") as fp:
                 label_run = fp.read().strip()
+                # if docker-run provided as multiline command
+                label_run = ' '.join(label_run.split('\\\n'))
```

Comment:
This looks fine to me. 

Although couldn't we change the top one to: ".rstrip()" instead of "strip()" (since that'll remove newlines) and then remove only backslashes instead?